src/consts/pipelinerun.ts

```diff
@@ -16,6 +16,8 @@ export enum PipelineRunLabel {
   COMMIT_TYPE_LABEL = 'pipelines.appstudio.openshift.io/type',
   REPOSITORY_NAME = 'pipelinesascode.tekton.dev/url-repository',
   SNAPSHOT = 'appstudio.openshift.io/snapshot',
+  COMMIT_EVENT_TYPE_LABEL = 'pipelinesascode.tekton.dev/event-type',
+  PULL_REQUEST_NUMBER_ANNOTATION = 'build.appstudio.redhat.com/pull_request_number',
```

Comment:
we can rely on core pac labels for pull request number, rather than the one which build service sets, because older pipelineruns might not have this label but core pac label will be always be set for sure.

```suggestion
  PULL_REQUEST_NUMBER = 'pipelinesascode.tekton.dev/pull-request',
```